### PR TITLE
contact-store: disallow data: urls from being added

### PR DESCRIPTION
**Pull request by tacryt-socryp**
Fix https://github.com/urbit/landscape/issues/675 for all new data, no migration needed.

### CODE REVIEW

File: pkg/arvo/app/contact-store.hoon [hunk 1]
```diff
@@ -126,6 +126,14 @@
               !=(contact(last-updated *@da) u.old(last-updated *@da))
           ==
         [~ state]
+      ~|  "cannot add a data url to cover!"
+      ?>  ?|  ?=(~ cover.contact)
```
Comment (liam-fitzgerald) [inline pkg/arvo/app/contact-store.hoon:130]: What's the point of this check?

Comment (tacryt-socryp) [reply pkg/arvo/app/contact-store.hoon:130]: Either there is no cover image OR we check to make sure the cover image does not begin with the data url schema

**Review by liam-fitzgerald — APPROVED** [review]

**Comment by ritfed-tabryg** [comment]
I imagine the rationale here was to prevent someone from setting a megabyte-size base64-encoded data URL as their profile pic, but it's unfortunate this blocks SVG profile images too, which can be pretty small. Would the Landscape team be open to allowing data URLs of sizes under, say, 4096 bytes?

**Comment by liam-fitzgerald** [comment]
@ritfed-tabryg there's nothing stopping you from setting an externally hosted SVG as a profile pic. The reason this was pulled out was due to concerns about memory pressure, so I'd defer to @joemfb about what sane default for potential image size should be.

**Pull request merged**